Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveElevatorToGivenHeightForTimeCommand.java

```diff
@@ -0,0 +1,59 @@
+package competition.subsystems.elevator.commands;
```

Comment:
This name makes me think that it has a movement timeout. Could we rename this to be "MoveElevatorAndHold(ForTime)Command"? Or perhaps "stabilize" instead of "hold"?

Reply:
Stabilize is really good.